src/Automation/Automation/Cmdlet/GetAzureAutomationModule.cs

```diff
@@ -43,21 +53,36 @@ protected override void AutomationProcessRecord()
             IEnumerable<Module> ret = null;
             if (!string.IsNullOrEmpty(this.Name))
             {
-                ret = new List<Module>
+                if (string.IsNullOrWhiteSpace(RuntimeVersion) || RuntimeVersion.Equals(Constants.RuntimeVersion.PowerShell51))
+                {
+                    ret = new List<Module>
                 {
                    this.AutomationClient.GetModule(this.ResourceGroupName, this.AutomationAccountName, this.Name)
                 };
```

Comment:
```suggestion
                    {
                       this.AutomationClient.GetModule(this.ResourceGroupName, this.AutomationAccountName, this.Name)
                    };
```
Suitable indents will make the code more readable

Reply:
Updated Thanks